src/Auth0.AuthenticationApi/Builders/AuthorizationUrlBuilder.cs

```diff
@@ -176,5 +176,25 @@ public AuthorizationUrlBuilder WithConnectionScopes(params string[] connectionSc
         {
             return WithConnectionScope(String.Join(" ", connectionScope));
         }
+
+        /// <summary>
+        /// Adds the `organization` query string parameter.
+        /// </summary>
+        /// <param name="organization">The ID of the organization to which the user wants to authenticate.</param>
```

Comment:
I would just keep this simpler and more concise:

```suggestion
        /// <param name="organization">The ID of the organization to log the user in to</param>
```